public/js/bookmarklet-popup.js

```diff
@@ -47,7 +47,7 @@ function bookmarklet_popupInit(host) {
     .appendTo('#bookmarklet_popup')
     .css({
       position: 'absolute',
-      right: 0, top: 0, width: '350px', height: '350px',
+      right: 0, top: 0, width: '350px', height: '420px',
```

Comment:
It seems like pixel-perfect positioning of the "Post" button is not really feasible with variable-height select2 input. Try to add 2 recipients, for example, and it will be shifted:

![image](https://cloud.githubusercontent.com/assets/1071933/13446226/f1a8bcd4-e011-11e5-87d5-c6c3b24da376.png)

Add another 2 recipients and it will be almost out of reach:

![image](https://cloud.githubusercontent.com/assets/1071933/13446238/09d8a71a-e012-11e5-9509-4d750ca5104e.png)

So we might need to either make select2 input have a fixed height, or add some additional room below it (i.e. make its _container_ fixed-height), or only display usernames in the list (instead of displayname+username), or some combination of these options. WDYT?